src/calendar/view/CalendarView.ts

```diff
@@ -470,8 +471,22 @@ export class CalendarView extends BaseTopLevelView implements TopLevelView<Calen
 			})
 		}
 
-		return calendarViewValues.map((viewType) =>
-			m(
+		return calendarViewValues.map((viewType) => {
+			const buttonAttrs = {
+				label: () => viewType.name,
+				icon: () => viewType.icon,
+				href: m.route.get(),
+				isSelectedPrefix: viewType.href,
+				colors: NavButtonColor.Nav,
+				// Close side menu
+				click: () => {
+					this._setUrl(viewType.value, this.viewModel.selectedDate())
+
+					this.viewSlider.focus(this.contentColumn)
+				},
+				disableHoverBackground: true,
+			}
+			return m(
 				".folder-row.flex.flex-row", // undo the padding of NavButton and prevent .folder-row > a from selecting NavButton
 				m(
 					".flex-grow.ml-negative-s",
```

Comment:
it still has `.ml-negative-s`, it was probably there because of the selection line, if you remove it it you can probably use `mlr-button`